src/Apps/Order/Routes/Shipping2/Components/FulfillmentDetails.tsx

```diff
@@ -319,40 +319,9 @@ const getInitialValues = (
 
   const savedAddresses = meData.addressList
 
-  // The default ship-to address should be the first one that
-  // can be shipped-to, preferring the default
-
-  const defaultUserAddress = getDefaultUserAddress(
+  return getInitialShippingValues(
     savedAddresses,
+    orderData.shipsFrom,
     orderData.availableShippingCountries
   )
```

Comment:
Moved the 'calculate initial shipping address from the order' into its own function.